yhondanilo/chess_programas.rb

```diff
@@ -0,0 +1,43 @@
+
+ arr=Array.new(3){Array.new(3)}
+  arr[0][0]=1
```

Comment:
Inconsistent indentation detected.<br>Surrounding space missing for operator '='.